packages/mask/src/plugins/Collectible/SNSAdaptor/MakeOfferDialog.tsx

```diff
@@ -74,6 +75,11 @@ export function MakeOfferDialog(props: MakeOfferDialogProps) {
     const { asset, open, onClose } = props
     const isAuction = asset?.value?.is_auction ?? false
     const isVerified = asset?.value?.is_verified ?? false
+    const leastPrice =
+        asset?.value && asset.value.orders?.length
+            ? new BigNumber(asset.value.orders[0].base_price ?? '0')
+            : new BigNumber('0')
```

Comment:
Use `ZERO` in packages/web3-shared/base/utils/number.ts.

Reply:
modified